framework/web/AssetManager.php

```diff
@@ -521,7 +527,13 @@ protected function publishDirectory($src, $options)
         if ($this->linkAssets) {
             if (!is_dir($dstDir)) {
                 FileHelper::createDirectory(dirname($dstDir), $this->dirMode, true);
-                symlink($src, $dstDir);
+                try { // fix #6226 symlinking multi threaded
+                    symlink($src, $dstDir);
+                } catch (\Exception $e) {
+                    if (!is_dir($dstDir)) {
```

Comment:
What if there was a directory with the same name you tried creating symlink with?